package.json

```diff
@@ -332,13 +332,15 @@
     "format": "prettier --write .",
     "pretest": "npm run compile && tsc -p ./client",
     "test": "node ./client/out/test/runTest.js",
+    "coverage": "c8 --clean --lines 75 npm run test",
```

Comment:
as there's a config file now, do we want to move --lines config into the config file?

Reply:
@scnwwu  thanks. Agreed. will add that to the .c8rc